app/models/item.rb

```diff
@@ -59,6 +59,10 @@ def average_fulfillment_time
     end
   end
 
+  def self.merchant_items(merchant)
```

Comment:
Wouldn't this be the same as calling merchant.items?

Reply:
Hmm... Yes, I think you're right.

Reply:
Yep, I'm removing this method from the pull request now to clean things up. Thanks for the suggestion!